src/Rabbitmq/Manager/Manager.php

```diff
@@ -212,6 +222,19 @@ public function fetchSource(string $connectionIdentifier) : \Maleficarum\Rabbitm
         
         return $connection;
     }
-    
+
+    /**
+     * Get connectionIdentifier based on current testMode.
+     *
+     * @param \Maleficarum\Command\AbstractCommand $command
+     * @param string $connectionIdentifier
+     * @return string
+     */
+    private function getConnectionIdentifier(\Maleficarum\Command\AbstractCommand $command, string $connectionIdentifier): string
+    {
```

Comment:
This opening bracket should be at the end of the previous line.

Reply:
fixed